package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dashevo/dapi-grpc",
-  "version": "0.12.1",
+  "version": "0.13.0-dev.1",
```

Comment:
We need to bump the version in a separate PR (chore) due to conventional changelog flow.